book/chapter-04.md

```diff
@@ -426,7 +426,7 @@ The highest-level summary is that XCAL provides a very good approximation to an
 
 The critical difference is that XCAL uses bidirectional activation dynamics to communicate error signals throughout the network, whereas backpropagation uses a biologically implausible procedure that propagates error signals backward across weight values, in the opposite direction of the way that activation typically flows (hence the name).  As discussed in the main chapter, the XCAL network experiences a sequence of activation states, going from an expectation to experiencing a subsequent outcome, and learns on the difference between these two states.  In contrast, backpropagation computes a single error *delta* value that is effectively the difference between the outcome and the expectation, and then sends this single value backwards across the weights.  In the following math, we show how these two ways of doing error-driven learning are approximately equivalent.
 
-The primary value of this exercise is to first establish that XCAL can perform a powerful, effective form of error-driven learning, and also to obtain further insights into the essential character of this error-driven learning by understanding how it is derived from first principles.  One of the most important intuitive ideas that emerges from this analysis is the notion of **credit assignment** --- you are encouraged to read up through that section (or just skip ahead to it if you really can't stand the following math, which again only requires basic algebra).
```

Comment:
removed this parenthetical because:
* Before it was really unclear where exactly the reader should skip to.
* With my edits, the first discussion of credit assignment is only a few paragraphs down, and there's not much math between here and there..